README.md

```diff
@@ -1,6 +1,11 @@
 # GovWifi Authentication API
 
-Authentication API for GovWifi
+This is the user authentication API for GovWifi.
+
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
```

Comment:
I'd in-line this link as it's only used once in this doc, think it's still clear

```suggestion
The private GovWifi [build repository][https://github.com/alphagov/govwifi-build] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
```

Reply:
The markdown linter suggests using no bare URLs.

Reply:
Interesting, that's not my understanding of what bare URLs are (is it [this rule](https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md)?)

Reply:
And it seems a lot of the repos we have and the GDS Way repo have it like this.

Reply:
Ah, OK, I can see the benefits of that style - having all the links defined at the bottom of the file so they're easy to update. 

Should we drag [build-repo] to the bottom of this file?